### ISSUE TITLE
create  workflow for containerstats

### ISSUE THREAD

**Issue by rajinish01** [nudjur/containerstats#9]

**Comment by sundarrajan78** [comment]
@rajinish01 on leave today

**Comment by rajinish01** [comment]
low priority now will come back to this.

**Comment by rajinish01** [comment]
no progress

**Comment by sundarrajan78** [comment]
@rajinish01 Can you propose the steps so that @kollaharshitha can have a look at this one next week?

**Comment by rajinish01** [comment]
Containerstats is written in python so we need to create 
- [ ] pypi workflow
- [ ] docker release workflow
- [ ] github release workflow

refer: https://github.com/nudjur/infra/blob/master/docs/checklist_for_new_repo.md

**Comment by sundarrajan78** [comment]
> Containerstats is written in python so we need to create
> 
> * [ ]  pypi workflow
> * [ ]  docker release workflow
> * [ ]  github release workflow
> 
> refer: https://github.com/nudjur/infra/blob/master/docs/checklist_for_new_repo.md

@kollaharshitha Can you embark on this tomorrow?

**Comment by sundarrajan78** [comment]
> > Containerstats is written in python so we need to create
> > 
> > * [ ]   pypi workflow
> > * [ ]   docker release workflow
> > * [ ]   github release workflow
> > 
> > refer: https://github.com/nudjur/infra/blob/master/docs/checklist_for_new_repo.md
> 
> @kollaharshitha Can you embark on this tomorrow?

@kollaharshitha Can you embark on this tomorrow?

**Comment by kollaharshitha** [comment]
will work on this now

**Comment by sundarrajan78** [comment]
@kollaharshitha Post update

**Comment by kollaharshitha** [comment]
will issue pr now.

**Comment by kollaharshitha** [comment]
https://github.com/nudjur/containerstats/pull/10

**Comment by kollaharshitha** [comment]
> #10

pr got merged

**Comment by kollaharshitha** [comment]
> > #10
> 
> pr got merged

@rajinish01 can we close this isssue?

**Comment by rajinish01** [comment]
yes @kollaharshitha did you check if github_release happened and new image pushed to docker?

**Comment by sundarrajan78** [comment]
@kollaharshitha post updates

**Comment by kollaharshitha** [comment]
Pr got merged and actions been in queued state from last 1hr.It's probably because the webhooks for that repository isn't configured yet. @ManishaBayya is checking that

**Comment by ManishaBayya** [comment]
Webhooks are configured to request self-hosted runner when action is started. The webhook events are sent properly.
![Screenshot from 2022-01-03 11-29-10](https://user-images.githubusercontent.com/34313380/147903218-6c2e92f9-96e3-4b43-9601-bc190b971b5a.png)

But the runner is not being allocated for the action. Hence the action was waiting for 5 mins as we have set for timeout minutes here - https://github.com/nudjur/containerstats/actions/runs/1640273315/workflow#L16 and then it was getting timed out and canceled.

Informed @kartikdc to check why the runner was not being allocated.
